ui.frontend/src/view/FormPanel.js

```diff
@@ -0,0 +1,71 @@
+/*******************************************************************************
+ * Copyright 2022 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+import {Constants} from "../constants";
+import FormFieldBase from "./FormFieldBase";
+
+export default class FormPanel extends FormFieldBase {
+
+    constructor(params) {
+        super(params);
+        this.children = [];
+    }
+
+    addChild(childView) {
+        this.children.push(childView);
+    }
+
+    /**
+     * applies full state of the field to the HTML. Generally done just after the model is bound to the field
+     * @param state
+     * @private
+     */
+    _applyState(state) {
```

Comment:
We should be following the new hash convention

Reply:
can you give some examples, this was recently added by @vdua